### PR TITLE
Commit two helper scripts 

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
Add two useful Python scripts:

1. _run/gen-explicit-fee-schedules.py_ to translate a Services _feeSchedules.json_ into the format used by the public pricing calculator.
2. _test-clients/yahcli/run/with-service-endpoints.py_ to augment a `(0.0.101, 0.0.102)` pair of deprecated-style yahcli JSON with the new `ServiceEndpoint` fields [as documented in the protobufs](https://hashgraph.github.io/hedera-protobufs/#proto.NodeAddress).

### CODE REVIEW

**Review by abhishek-hedera — APPROVED** [review]
LGTM

File: run/gen-explicit-fee-schedules.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        for typed_prices in prices_list:
+            this_type = typed_prices.get('subType', 'DEFAULT')
+            this_type_prices = {}
+            for provider in providers:
+                this_type_prices[provider] = typed_prices[provider]
+            prices_by_type[this_type] = this_type_prices
+        typed_schedules[function] = prices_by_type
+
+with open('typedFeeSchedules.json', 'w') as fout:
+    json.dump(typed_schedules, fout, indent=2)
```
Comment (ljianghedera) [inline run/gen-explicit-fee-schedules.py:29]: Nice move! I was thinking to consolidate the server side feeSchedules.json and client side FeeSchedule.json to be one file so we will DRY!

This is a good solution!

**Review by ljianghedera — APPROVED** [review]
Awesome!

**Review by anighanta — APPROVED** [review]
Thanks @tinker-michaelj. I remember the days testing the yacli tool with different address book versions manually modifying fields for each version :(

**Pull request merged**